app/models/swimming_pool.rb

```diff
@@ -0,0 +1,7 @@
+class SwimmingPool < ApplicationRecord
+  belongs_to :user
+  has_many :bookings
+  validates :capacity, presence: true
```

Comment:
We could refactor this part:
validates :capacity, :location, :size, presence: true